### PR TITLE
Feature/`flask import_data` Command

### PR DESCRIPTION
**Pull request by mblomdahl**
This PR adds a giant "import_data" hack to Flask commands. It's not pretty, at all. But it should allow us to easily import data until January 2nd launch.

How to use the import:

    export FLASK_APP=autoapp
    export FLASK_DEBUG=1
    flask db upgrade
    flask import_data

### CODE REVIEW

**Comment by mxtthias** [comment]
Good job! A few comments:

* Especially `_generate_xl_auth_cataloging_admins_and_collections` can benefit from some comments, outlining the different steps.
* There are a bunch of `print()` and `click.echo()` that only print a variable with no additional information, which I assume makes debugging a bit trickier.
* There's a line (218 in `commands.py`) that is commented out. Remove?
* There's a TODO (335 in `commands.py`) that I don't understand. Who is it for and that is `me` in this context?
* Is `requests` necessary in `_get_voyager_data`? To me it looks like we're reading a file that the code could read directly from disc instead. Or is it to make all data access consistent?

File: xl_auth/commands.py [hunk 1]
```diff
@@ -172,3 +172,254 @@ def urls(url, order):
...
+                    continue
+
+                for voyager_collection in voyager_sigel_to_collections[sigel]:
+                    if voyager_collection not in xl_auth_collections:
+                        try:
+                            xl_auth_collections[voyager_collection] = \
+                                _get_collection_details_from_bibdb(voyager_collection)
+                        except AssertionError:
+                            voyager_sigels_unknown_in_bibdb.add(voyager_collection)
+                            continue  # TODO: Review me!
```
Comment (mxtthias) [inline xl_auth/commands.py]: I don't understand the TODO. Who is it for and that is `me` in this context?

File: xl_auth/commands.py [hunk 2]
```diff
@@ -172,3 +172,254 @@ def urls(url, order):
...
+        }
+
+    def _get_bibdb_sigels_not_in_voyager(bibdb_sigels, voyager_sigels):
+        unknown_sigels = set()
+        for bibdb_sigel in bibdb_sigels:
+            if bibdb_sigel not in voyager_sigels:
+                unknown_sigels.add(bibdb_sigel)
+        return unknown_sigels
+
+    def _generate_xl_auth_cataloging_admins_and_collections(bibdb_cataloging_admin_to_sigels,
```
Comment (mxtthias) [inline xl_auth/commands.py]: There are a bunch of `print()` and `click.echo()` statements here that only print a variable, making the output difficult to interpret. Can you add some more info there?

This (and maybe some other functions too) can benefit from some comments, outlining the different steps.

File: xl_auth/commands.py [hunk 3]
```diff
@@ -172,3 +172,254 @@ def urls(url, order):
...
+                    raise AssertionError('Duplicate results for sigel %r' % code)
+                bibdb_api_data = chunk
+        if not bibdb_api_data:
+            raise AssertionError('Zero results for sigel %r' % code)
+
+        if bibdb_api_data['type'] in {'library', 'bibliography'}:
+            category = bibdb_api_data['type']
+        else:
+            category = 'uncategorized'
+        # assert bibdb_api_data['country_code'] == 'se'
```
Comment (mxtthias) [inline xl_auth/commands.py]: Remove?

File: xl_auth/commands.py [hunk 4]
```diff
@@ -172,3 +172,254 @@ def urls(url, order):
...
+        }
+
+        if bibdb_api_data['date_created']:
+            collection['created_at'] = \
+                dt.datetime.strptime(bibdb_api_data['date_created'], '%Y-%m-%dT%H:%M:%S')
+
+        return collection
+
+    def _get_voyager_data():
+        raw_voyager_sigels_and_locations = requests.get(
```
Comment (mxtthias) [inline xl_auth/commands.py]: Is `requests` necessary here? It feels like we should be able to read that file from disc instead. Or is this to keep our data access consistent?

File: migrations/versions/564f2b578774_add_cataloguing_admin_to_permissions_.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-"""Add cataloguing_admin to permissions table.
+"""Add catalogue_admin to permissions table.
```
Comment (mxtthias) [inline migrations/versions/564f2b578774_add_cataloguing_admin_to_permissions_.py]: Should we change the file name while we're at it?

File: xl_auth/commands.py (hunk 4 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: Not necessary, but I thought it was more straight-forward loading it from network (like we do with BibDB data). And it avoids ...
1. figuring out a good place to put this blob-like file of 4000 lines in the repo, maybe a "/resources" dir?
2. slippery-slope of checking in misc "good to have" stuff!

File: xl_auth/commands.py (hunk 1 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: `me` would be the *continue* statement. 😄 Is it needed or not? I think not... Removing it.

Comment (mblomdahl) [reply xl_auth/commands.py]: Interesting thing.. When replying to this comment on *Files Changed* tab, the comment does not show up here. Your comments are in both..

File: xl_auth/commands.py (hunk 4 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: Not necessary, but I thought it was more straight-forward loading it from network (like we do with BibDB data). And it avoids ...
1. figuring out a good place to put this blob-like file of 4000 lines in the repo, maybe a "/resources" dir?
2. slippery-slope of checking in misc "good to have" stuff!

File: xl_auth/commands.py (hunk 1 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: `me` would be the *continue* statement. 😄 Is it needed or not? I think not... Removing it.

**Comment by mblomdahl** [comment]
@mxtthias Interesting thing.. When replying to this comment on *Files Changed* tab, the comment does not show up here.

File: xl_auth/commands.py (hunk 2 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: Done!

File: xl_auth/commands.py (hunk 1 above)
Comment (mblomdahl) [reply xl_auth/commands.py]: Correction: It IS needed.

**Pull request merged**